### PR TITLE
Get rid of sample-entities command. Provide option to filter by secret and casefile flags when listing collections.

### PR DESCRIPTION
**Pull request by sunu**
Doing the sampling inside ES led to hanging queries. So we do the sampling outside ES.

The other alternative is to do the sampling inside ftm-store database but we have to additionally implement property based filters in SQL in that case.

### CODE REVIEW

**Comment by pudo** [comment]
Hey if we're getting rid of the `randomize=True` flag in `iter_`, can we also delete it upstream? Otherwise it just sticks around as cruft.

**Comment by sunu** [comment]
Oh I should mark this PR as WIP. Let's not merge this yet. I'm still trying to figure out if it holds good for the bigger stream or not. Will clean it up after that.

**Comment by sunu** [comment]
@mynameisfiber Should we get rid of `sample-entities` command now that we sample from ftm-store or is it still useful to keep?

**Comment by mynameisfiber** [comment]
@sunu delete it!

**Comment by pudo** [comment]
@mynameisfiber please can you handle?

**Comment by mynameisfiber** [comment]
@pudo yup

**Comment by mynameisfiber** [comment]
@sunu @pudo updated

File: aleph/index/entities.py [hunk 1]
```diff
@@ -66,19 +64,31 @@ def iter_entities(
     includes=PROXY_INCLUDES,
     excludes=None,
     filters=None,
-    should=None,
     sort=None,
-    raise_on_error=True,
+    randomize=False,
+    random_seed=None,
```
Comment (sunu) [inline aleph/index/entities.py]: @mynameisfiber May be we should get rid of these arguments too since they are not used anywhere anymore?

**Comment by mynameisfiber** [comment]
@sunu good catch!

**Comment by mynameisfiber** [comment]
anyone mind if i squash all these commits and resolve the merge conflict?

**Comment by sunu** [comment]
@mynameisfiber no, please go ahead!

**Comment by mynameisfiber** [comment]
@sunu all cleaned up

**Comment by pudo** [comment]
Lovely! Thanks, guys!

**Pull request merged**